services/uploads/serverless.yml

```diff
@@ -115,17 +116,57 @@ resources:
                 - DELETE
                 - HEAD
               MaxAge: 3000
+              ExposedHeaders:
+                - ETag
         NotificationConfiguration:
           LambdaConfigurations:
             - Event: s3:ObjectCreated:Put
               Function: !GetAtt AvScanLambdaFunction.Arn
       DependsOn: LambdaInvokePermission
+    AVBucketPolicy:
```

Comment:
This policy shouldn't be needed, as the lambda execution role (see the provider.iam.role section) allows access to the bucket.
This policy does specify one action that the provider.iam.role does not... "s3:GetBucketLocation".  So if "s3:GetBucketLocation" is actually needed, add that Action to the provider.iam.role statements above.